tests/console/snapper_create.pm

```diff
@@ -20,7 +20,14 @@ use utils;
 # that we got what we expect. See poo#25716
 sub get_last_snap_number {
     # get snapshot id column, tail before head to avoid SIGPIPE
-    my $snap_head         = script_output("snapper list | tail -n +1 | head -n1");
+    my $snap_head = script_output("snapper list | tail -n +1 | head -n1");
+
+    # strip kernel messages - for some reason we always get something like this at this very position:
+    # [ 1248.663412] BTRFS info (device vda2): qgroup scan completed (inconsistency flag cleared)
+    my @lines = split(/\n/, $snap_head);
+    @lines = grep(/\|/, @lines);
+    $snap_head = $lines[0];
```

Comment:
To avoid accessing non-existing element, don't you mind adding a check to assure that it was found? Otherwise test will fail with an exception, which is not straight forward.

Reply:
Something like this?

Reply:
Yes, and you can actually just write `unless @lines`, as empty array denotes false in perl.